package.json

```diff
@@ -61,10 +65,19 @@
     "eslint": "^3.3.1",
     "eslint-config-babel": "^1.0.1",
     "eslint-plugin-babel": "^3.3.0",
-    "eslint-plugin-flow-vars": "^0.5.0"
+    "eslint-plugin-flow-vars": "^0.5.0",
+    "mocha": "^3.0.2"
   },
   "babel": {
     "plugins": [
+      "babel-plugin-transform-es2015-destructuring",
```

Comment:
I ❤️ writing in ES-next and this is the basic set of plugins missing from Node 4.x. If you'd prefer I dev with Node 6 and take this out or some other thing, I'm happy to do so.